src/core/ledger/ledger.js

```diff
@@ -778,6 +786,10 @@ export class Ledger {
     account.paid = G.add(params.grainReceipt.amount, account.paid);
     account.allocationHistory.push(params);
     account.balance = G.add(params.grainReceipt.amount, account.balance);
+    this._totalCirculatingGrain = G.add(
+      this._totalCirculatingGrain,
+      params.grainReceipt.amount
+    );
```

Comment:
Did you mean to bake this into this PR? If so, please update the description to acknowledge it.

Reply:
Nope, thanks!